app/web/src/store/func/funcs.store.ts

```diff
@@ -955,22 +955,22 @@ export const GenerateAwsFunctionKinds = {
   AssetSchema: {
     description: "AWS Asset Schema",
     exampleCommand: {
```

Comment:
I don't mind this nomenclature (it sounds more correct!), but there are a lot of places we use the command/subcommand structure; we'll need to go through and fix the others. There's a type called AwsCommand, I believe, that could help fix the stuff on the AWS side.

Not necessary for this PR though, perfect being the enemy of the good and all that :)

Reply:
Service and command *are* the correct terminolgy here - it's what AWS specifies. The internals of command and subcommand should not leak out